stratum/hal/lib/barefoot/bf_pal_wrapper.cc

```diff
@@ -280,6 +282,46 @@ ::util::Status BFPalWrapper::PortLoopbackModeSet(int unit, uint32 port_id,
   return ::util::OkStatus();
 }
 
+namespace {
+
+// PortKey uses three possibile values for channel:
+//   > 0: port is channelized (first channel is 1)
+//   0: port is not channelized
+//   < 0: port channel is not important (e.g. for port groups)
+int GetZeroBasedChannel(const PortKey* port_key) {
```

Comment:
We can use reference here, like `const PortKey& port_key` so we don't need to pass the address to this function.
and replace `->` to just `.`

Reply:
Makes sense. Thanks!